<!DOCTYPE html>
<html lang="zh">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>
      
    Week 10 Preview - Yuanye Chi's Blog
    
    </title>
    <link rel="shortcut icon" href="https://i.loli.net/2019/02/20/5c6cf0555ffe6.jpeg" type="image/png" />

    
    
    <link href="atom.xml" rel="alternate" title="Yuanye Chi's Blog" type="application/atom+xml">
    <link rel="stylesheet" href="asset/css/style.min.css">
    <link rel="stylesheet" href="asset/css/doc.css">
    <script src="asset/app.js"></script>
</head>
  <body>
    <section class="hero">
      <div class="hero-head">
          <nav class="navbar" role="navigation" aria-label="main navigation">
              <div class="container">
              <div class="navbar-brand">
                
                <a target="_self" class="navbar-item " href="index.html">Home</a>
                
                <a target="_self" class="navbar-item " href="archives.html">Archives</a>
                
                <a target="_self" class="navbar-item " href="about.html">About</a>
                
                <a target="_self" class="navbar-item " href="https://chiyuanye-com-old-site.onrender.com/">OldSite</a>
                

                <a role="button" id="navbarSNSRssSwitchBtn" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navbarSNSRssButtons">
                  <span aria-hidden="true"></span>
                  <span aria-hidden="true"></span>
                  <span aria-hidden="true"></span>
                </a>
              </div>
            
              <div id="navbarSNSRssButtons" class="navbar-menu">
                <div class="navbar-start">
                  
                </div>
            
                <div class="navbar-end">
                  <div class="navbar-item">
                    <!--buttons start-->
                    <div class="buttons">
                      
                        
                        
                        
                        <a href="https://github.com/tsiye" target="_blank" title="github">
                            <span class="icon is-large has-text-grey-darker">
                               <svg class="svg-inline--fa fa-github fa-w-16 fa-lg" aria-hidden="true" focusable="false" data-prefix="fab" data-icon="github" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512" data-fa-i2svg=""><path fill="currentColor" d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"></path></svg><!-- <i class="fab fa-github fa-lg"></i> -->
                            </span>
                          </a>
                        
                        
                      
                      <a href="atom.xml" target="_blank" title="RSS">
                          <span class="icon is-large has-text-black-bis">
                              <svg class="svg-inline--fa fa-rss fa-w-14 fa-lg" aria-hidden="true" focusable="false" data-prefix="fas" data-icon="rss" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" data-fa-i2svg=""><path fill="currentColor" d="M128.081 415.959c0 35.369-28.672 64.041-64.041 64.041S0 451.328 0 415.959s28.672-64.041 64.041-64.041 64.04 28.673 64.04 64.041zm175.66 47.25c-8.354-154.6-132.185-278.587-286.95-286.95C7.656 175.765 0 183.105 0 192.253v48.069c0 8.415 6.49 15.472 14.887 16.018 111.832 7.284 201.473 96.702 208.772 208.772.547 8.397 7.604 14.887 16.018 14.887h48.069c9.149.001 16.489-7.655 15.995-16.79zm144.249.288C439.596 229.677 251.465 40.445 16.503 32.01 7.473 31.686 0 38.981 0 48.016v48.068c0 8.625 6.835 15.645 15.453 15.999 191.179 7.839 344.627 161.316 352.465 352.465.353 8.618 7.373 15.453 15.999 15.453h48.068c9.034-.001 16.329-7.474 16.005-16.504z"></path></svg><!-- <i class="fas fa-rss fa-lg"></i> -->
                          </span>
                      </a>
                    </div>
                    <!--buttons end-->

                  </div>
                </div>
                </div>
              </div>
            </nav>
      </div>

 <div class="hero-body ct-body"></div>
      
    </section>
    <section class="ct-body">
      <div class="container">
          <div class="columns is-variable bd-klmn-columns is-4 is-centered">
              <div class="column is-four-fifths">
                  <div class="post-body single-content">
                    
                    <h1 class="title">
                            Week 10 Preview   
                      </h1>
                     
                    
                      <div class="media">
                            
                            <figure class="media-left">
                              <p class="image is-48x48">
                                
                                  <img class="is-rounded" src="https://i.loli.net/2019/02/20/5c6cf0e7a42f9.jpg">
                                
                              </p>
                            </figure>
                            
                            <div class="media-content">
                              <div class="content">
                                <p>
                                 <span class="date">2022/11/06 11:06 AM</span>
                                  <span class="tran-posted-in">posted in</span>&nbsp; 
                                  
                                      <span class="posted-in"><a href='BIO105.html'>BIO105</a></span>
                                         
                                  
                                    &nbsp;&nbsp;<a href="16677507862746.html#disqus_thread"><span class="tran-disqus-comments">comments</span></a>
                                  

                                   
                                      
                                  <br />
                                  <span class="tran-tags">Tags:</span>&nbsp;
                                  
                                    <a class="tag is-link is-light" href='tag_BIO-105.html'>#BIO-105</a>
                                     

                                </p>
                              </div>
                            </div>
                         
                    </div>
                </div>
                  <article class="markdown-body single-content">
                    <blockquote>
<p>This week,examine eukaryotes process newly-transcribed mRNAs and how the different steps of mRNA processing are regulated. The splicing work and how alternative splicing generates amazing protein diversity.<br />
A special treat: a medical case study that illustrate how aberrant splicing can lead to human disease and possible ways to treat it.</p>
</blockquote>
<h1><a id="chromatin-and-transcription" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Chromatin and Transcription</h1>
<blockquote>
<p>Specific in eukaryotes, because bacteria do not have chromatins.</p>
</blockquote>
<h2><a id="chromatin-can-be-modified-in-several-ways-to-regulate-gene-expression" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Chromatin can be modified in several ways to regulate gene expression</h2>
<ol>
<li>Nucleosome composition(HH1 and variant histones)
<ol>
<li>They can have different effects on chromatin and gene expression</li>
</ol>
</li>
<li>Nucleosome remodeling complexes
<ol>
<li>Bind to or either remove, add or migrate nucleosome throughout the genome to allow for different gene expression.</li>
</ol>
</li>
<li>Histone and DNA modification
<ol>
<li>Modify histone and DNA with various chemical modifications that result in downstream effects on transcription and gene expression.</li>
</ol>
</li>
</ol>
<h3><a id="nucleosomes-can-be-movedexchanged-by-chromatin-remodeling-enzymes" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Nucleosomes can be moved/exchanged by chromatin remodeling enzymes</h3>
<p><img src="media/16677507862746/16677654086785.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<blockquote>
<p>Use ATP hydrolysis to do these things.</p>
</blockquote>
<h4><a id="one-model-for-nucleosome-repositioning" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>One model for nucleosome repositioning</h4>
<p><img src="media/16677507862746/16677656927215.jpg" alt="" class="mw_img_center" style="width:450px;display: block; clear:both; margin: 0 auto;" /><br />
DNA painted orange is wrapped around by histone can be moved so the DNA will be exposed.</p>
<h2><a id="histone-and-dna-modification" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Histone and DNA modification</h2>
<h3><a id="histone-tails" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Histone tails</h3>
<h4><a id="histone-tails-extrude-far-from-the-nucleosome" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Histone tails extrude far from the nucleosome</h4>
<p><img src="media/16677507862746/16677658930243.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<h3><a id="histone-tails-can-be-modified-in-several-ways" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Histone tails can be modified in several ways</h3>
<p><img src="media/16677507862746/16677659883305.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /><br />
<em>Triangle here represents the nucleosome core.</em></p>
<p>The modification number on N-terminal tail is much more than it on C-termminal tail.<br />
<img src="media/16677507862746/16677660060832.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<ul>
<li>Phosphorylation: S, T, Y</li>
<li>Acetylation: K</li>
<li>Methylation: K, R</li>
<li>Ubiquitination: K</li>
</ul>
<h3><a id="chemical-structure-of-histone-modifications" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Chemical Structure of Histone Modifications</h3>
<ol>
<li>
<p>Acetylation of Lys residues<br />
<img src="media/16677507862746/16677663018727.jpg" alt="" class="mw_img_center" style="width:200px;display: block; clear:both; margin: 0 auto;" /><br />
<strong>Acetyl group neutralizes(+) the positive charge of lysine</strong>. In doing so, <em>decrease the attraction of the negatively charged DNA to the positively charged lysine side chain.</em> So the acetylation <strong>opens up the chromatin structure</strong>. Makes it easier to access the DNA.</p>
</li>
<li>
<p>Methylation of Lys residues.<br />
<img src="media/16677507862746/16677665386001.jpg" alt="" class="mw_img_center" style="width:300px;display: block; clear:both; margin: 0 auto;" /></p>
</li>
<li>
<p>Methylation of Arg residues<br />
<img src="media/16677507862746/16677666829377.jpg" alt="" class="mw_img_center" style="width:300px;display: block; clear:both; margin: 0 auto;" /><br />
<strong>Methylation has no effect on charge.</strong> So the impact of methylation is very complex. Sometimes result in open chromatin structure and somestimes result in close chromatin structure. Depends on protein interact with methylation.</p>
</li>
<li>
<p>Phosphorylation of Ser, Thr, and Tyr residues.<br />
<img src="media/16677507862746/16677667553908.jpg" alt="" class="mw_img_center" style="width:400px;display: block; clear:both; margin: 0 auto;" /><br />
<strong>Add (-) charge</strong>. Most of the time will further open up the chromatin structure, result in active gene expression.</p>
</li>
<li>
<p>Ubiquitin is a small polypeptide that can covalently attach to other proiteins(like histones) to change their function<br />
<img src="media/16677507862746/16677669527251.jpg" alt="" class="mw_img_center" style="width:400px;display: block; clear:both; margin: 0 auto;" /><br />
There are two lysines on ubiquitin protein where other ubiquitins can be conjugated.<em>poly-ubiquitylation</em> So ubiuqitin can help reruit other proteins to modification.</p>
</li>
</ol>
<h3><a id="histone-modifications-can-cooperate-to-establish-a-chromatin-state" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Histone Modifications can cooperate to establish a chromatin state</h3>
<blockquote>
<p>The state can be active or repressive.</p>
</blockquote>
<p><img src="media/16677507862746/16677674149322.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<p>If Ser10 is phosphorylated first, an activated state is established. In which Lys 14 on histone H3 will be acetylated and Lys9 will be prevented for having a repressive methyl group ligated to that.</p>
<p>If Ly9 is methylated first, a repressive state is established, which will prevent phosphate of Ser10, then prevent acetylation of Lys14.</p>
<p><strong>Depending on which happened first, we can have either activation or repressive transcription.</strong></p>
<h3><a id="enzymes-that-catalyze-histone-modifications" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Enzymes that catalyze histone modifications</h3>
<p><img src="media/16677507862746/16677679582738.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<ul>
<li>Open chromatin: bromodomain protein. binds to Ac-K</li>
<li>Closed chromatin: chromodomain protein. binds to Me-K</li>
</ul>
<h3><a id="several-processes-canc-synergize-to-regulate-transcriptional-activation" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Several Processes canc  Synergize to Regulate Transcriptional Activation</h3>
<blockquote>
<p>From complete repressive state to actvie state.</p>
</blockquote>
<p><img src="media/16677507862746/16677686850845.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<p>We start the repressive state. TATA box, place have eukaryotes transcription factors binding to initiate transcription. We want to expose this, to get out of the repressive environment.<br />
<img src="media/16677507862746/16677687162712.jpg" alt="" class="mw_img_center" style="width:450px;display: block; clear:both; margin: 0 auto;" /><br />
We are going to recuit a histone acetyle transcripse(HAT) called Gcn5, which will acetylate H3 and H4 by acetylated the lysine 9 and lysine 8.</p>
<p>Then acetyle group called protein kinase will phosphorylate other proteins. In doing so, it will recruit additional HAT that can engaging second acetylation event.<br />
So we phosphorylate on H3 Ser10 caused lysine 14 being acetylated.<br />
<img src="media/16677507862746/16677690507611.jpg" alt="" class="mw_img_center" style="width:450px;display: block; clear:both; margin: 0 auto;" /></p>
<p>Now we start to open up this chromatini structure, which will allow the recuitment of chromatin remodeling enzymes. (SWI/SNF) to move the nucleosome.<br />
<img src="media/16677507862746/16677691174989.jpg" alt="" class="mw_img_center" style="width:300px;display: block; clear:both; margin: 0 auto;" /></p>
<p>In doing so, once the nucleosome has been moved, which will expose the TATA box, allows for TFIIID and all other  core transcription factors to bind, which can then result in active transcription if immediator protein or additional ltranscription enhancers are found here.</p>
<p><img src="media/16677507862746/16677691392328.jpg" alt="" class="mw_img_center" style="width:300px;display: block; clear:both; margin: 0 auto;" /></p>
<p><strong>Without modifying the protein, we can directly modify DNA itself.</strong> This <em>usually cause a repressive effect.</em></p>
<h2><a id="dna-methylation-also-represses-gene-expression" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>DNA methylation also represses gene expression</h2>
<p>Start from active transcription witth activators and mediator.<br />
After dissociation of activators, methylation will happen on cytosine at CpG(cytosine of guanine) islands. Such methylation will turn off transcription.   Also further repress transcription by recruit histone deacetylase and chromatin remodeling complex.</p>
<p><img src="media/16677507862746/16677703536890.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<p>Mutation in the above process will cause diseases. One mutation is called MeCP2.<br />
Lossing methyl binding protein MeCP2, typically find in females. Because gene encode MeCP2 is on X chromosome. Male has such mutation will die before birth.</p>
<h1><a id="rna-processing-in-eukaryotes" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>RNA Processing in Eukaryotes</h1>
<h2><a id="mrna-processing-is-different-in-bacteria-vs-eukaryotes" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>mRNA processing is different in bacteria vs. eukaryotes</h2>
<p><img src="media/16677507862746/16677711058591.jpg" alt="" class="mw_img_center" style="width:450px;display: block; clear:both; margin: 0 auto;" /><br />
In bacteria, <em>transcription and translation happen simultaneously.</em> Because there is no physical separation.(nucleous)</p>
<p><img src="media/16677507862746/16677711854186.jpg" alt="" class="mw_img_center" style="width:450px;display: block; clear:both; margin: 0 auto;" /></p>
<ul>
<li>mRNA is transcribed in nucleus.</li>
<li>Various processing events happen including 5' cap.  <em>Removal of the introns and addition of polyA tail.</em></li>
<li>Then protein helps mRNA export to cytoplasm.</li>
<li>Ribosome binds to 5' cap to start translation.</li>
</ul>
<p>In both cases, The amout of protein produced can be regulated through the amount of mRNA degradation.</p>
<h2><a id="the-ctd-tail-of-rna-pol-ii-recruits-rna-processing-enzymes" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>The CTD tail of RNA pol II recruits RNA processing enzymes</h2>
<p>CTD:carboxyl terminal domain.<br />
The phosphorylation state dictate what processing factors is going to be recruited.<br />
<img src="media/16677507862746/16677720260827.jpg" alt="" class="mw_img_center" style="width:600px;display: block; clear:both; margin: 0 auto;" /></p>
<p><img src="media/16677507862746/16677720712415.jpg" alt="" class="mw_img_center" style="width:600px;display: block; clear:both; margin: 0 auto;" /></p>
<p><strong>During promoter escape, Ser 5 gets phosphorylated by TFIIH kinase</strong>.<br />
<strong>TFIIH kinase is really important in translation of eukaryotes.</strong> Then will add cap at 5' of mRNA.</p>
<p>During elongation, P-TEFb will phosphorylate Ser 2. <em>phosphotase will remove phosphate on ser 5.</em>  This helps to recruit splicing and other elongation factors.</p>
<h3><a id="a-7-methyl-guanine-cap-is-added-to-the-5-end-of-the-mrna" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>A 7-methyl guanine cap is added to the 5' end of the mRNA.</h3>
<blockquote>
<p>The first stage of eukaryote mRNA processing.</p>
</blockquote>
<p><img src="media/16677507862746/16677726477321.jpg" alt="" class="mw_img_center" style="width:400px;display: block; clear:both; margin: 0 auto;" /></p>
<p>7-Methyl-guanasine is linked to the rest of mRNA through <em>5',5'-Triphosphate linkage.</em>  which will</p>
<ul>
<li>Prevents degradation by exonucleases</li>
<li>Needed for proper splicing and translation</li>
</ul>
<p><img src="media/16677507862746/16677727866715.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /><br />
The enzyme doing this is called <strong>Guanylyltransferase</strong>, which is associated with Pol II CTD to ensure that each mRNA is capped as it is transcribed.</p>
<p>Once the cap is complete, the guanylyltransferase dissociates and the <strong>cap-binding complex(CBC) binds.</strong></p>
<h3><a id="polyadenylation-and-termination-are-tightly-linnked-in-eukaryotes" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Polyadenylation and termination are tightly linnked in eukaryotes</h3>
<blockquote>
<p>At the end of mRNA transcription<br />
Also dictated by phosphrolation on CTD of Pol II.</p>
</blockquote>
<p><img src="media/16677507862746/16677731140719.jpg" alt="" class="mw_img_center" style="width:450px;display: block; clear:both; margin: 0 auto;" /></p>
<p>When Pol II is transcribing through the polyA addition site.<br />
Consensus sequence:</p>
<ul>
<li>5' AAUAAA is 10-30 nt upstream of cleavage site.</li>
<li>GU-rich sequence 20-40 nt downstream of cleavage site.</li>
</ul>
<p>These two consensus sequences position the <strong>polyadenylation factor</strong> to bind to polyA signal, indicating mRNA cleavate. Also the whole complex will cause the <strong>endonuclease</strong> to cleave mRNA between these two different consensus sequences. <strong>PolyA polymerase(PAP)</strong> will synthesize a polyA tail(80-250 As) right downstream the mRNA been cleaved. <strong>PolyA binding protein</strong> will binds to this tail to protect the 3' of mRNA from degradation.</p>
<p>Worth mentioning is pol II will continue transcribing after this cleave action. There are other action to terminate its action.</p>
<h2><a id="summary-rna-pol-ctd-coordinates-transcription-and-pre-mrna-processing" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Summary: RNA pol CTD coordinates transcription and pre-mRNA processing</h2>
<p><img src="media/16677507862746/16677745588153.jpg" alt="" class="mw_img_center" style="width:550px;display: block; clear:both; margin: 0 auto;" /></p>
<p>After all these stuff, result in a <strong>capped and polyA tailed mRNA.</strong></p>
<p>However, the sequence of mRNA is not always determined by the sequence of DNA.<br />
There is process called <strong>RNA editing</strong>, which can change primary sequence of  mRNA after it is been transcribed.</p>
<h2><a id="rna-editing-a-way-to-change-the-coding-potential-of-genes" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>RNA editing: a way to change the coding potential of genes</h2>
<h3><a id="example-1-a-to-i-editing-by-adenosine-deaminase-adar" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Example 1: A-to-I editing by adenosine deaminase(ADAR)</h3>
<p><img src="media/16677507862746/16677748415344.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<p><img src="media/16677507862746/16677748851178.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /><br />
Indosine here is going to change the coding potential of mRNA. <strong>Gln -&gt; Arg.</strong></p>
<p>Glutamate receptor channel allows for calcium signaliing in the brain.</p>
<p>ADAR-mediated editing results in dramatical altering the channel properties.</p>
<h3><a id="example-2-c-to-u-editing-by-cytosine-deaminases-and-apobecs" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Example 2: C-to-U editing by cytosine deaminases and APOBECs</h3>
<p><img src="media/16677507862746/16677751120563.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /><br />
<img src="media/16677507862746/16677751413536.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /><br />
RNA editing create an early stop codon that results in a shorter from of ApoB.<br />
This form assists with the absorption of lipids in the intestine.</p>
<p>How the processed mRNA find their ways outside the nucleus into the cyroplasm in eukaryote cells.</p>
<h2><a id="movement-of-mrnas-into-and-out-of-the-nucleus" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Movement of mRNAs into and out of the nucleus</h2>
<blockquote>
<p>Primary mRNA will be processed and spliced.<br />
EJC(exon junction complex)</p>
</blockquote>
<p><img src="media/16677507862746/16677754903580.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<ul>
<li>EJCs are left behind on the processed mRNA following splicing</li>
<li>EJCs and the cap-binding complex(CBC) are important for mRNA transport out of the nucleus.</li>
</ul>
<p><img src="media/16677507862746/16677757770671.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<p>CBC is at the end of CTD tail. SR act with splicesome to promote mRNA splicing. Cap-binding complex will recruit addtional proteins together with EJC allow for the binding and transport of mRNA from nucleus into cytoplasm.</p>
<h1><a id="splicing" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Splicing</h1>
<blockquote>
<p>Objective:</p>
<ol>
<li>Identify consensus sequence that are important for splicing in eukaryotes</li>
<li>How RNA and proteins add together to carry out this reaction.</li>
</ol>
</blockquote>
<h2><a id="how-splicing-first-been-discovered-r-loop-analysis" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>How Splicing First Been Discovered? -- R-loop Analysis</h2>
<p>Electron microscopy provides evidence for the existence of introns</p>
<p><img src="media/16677507862746/16677773079542.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<p>The loops are the introns.</p>
<h3><a id="intron-numbers-increase-with-complexity-of-gemones" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Intron numbers increase with complexity of gemones</h3>
<p><img src="media/16677507862746/16677781061862.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<h2><a id="overview-of-eukaryotic-mrna-splicing-and-processing" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Overview of Eukaryotic mRNA splicing and processing</h2>
<blockquote>
<p>Splicing happens co-transcriptionally!!!</p>
</blockquote>
<p><img src="media/16677507862746/16677782064482.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<h2><a id="consensus-sequences-that-are-critical-for-splicing" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Consensus sequences that are critical for splicing</h2>
<blockquote>
<p>GU ---------- AG is the most highly conserved consensus sequences<br />
Unpaired A in th emiddle of a pyridine-rich region</p>
</blockquote>
<p><img src="media/16677507862746/16677783810058.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<h2><a id="overview-of-splicing-2-transesterification-reactions" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Overview of splicing: 2 Transesterification Reactions</h2>
<p><img src="media/16677507862746/16677796815797.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<ol>
<li>The branch-point 2'-OH attacks the 5' splice site.</li>
<li>The 5' splice site is now activated to attack the 3' splice site.</li>
<li>The intron is released from the spliced mRNA as a lariat.</li>
</ol>
<h3><a id="transesterification-1" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Transesterification 1</h3>
<p>2'-OH attacks the 5' splice site.</p>
<h3><a id="transesterification-2" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Transesterification 2</h3>
<p>3'OH of exon 1 attacks 3' splice site of intron</p>
<h3><a id="helper-complex-the-spliceosome-is-composed-of-rna-and-protein" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Helper Complex: The spliceosome is composed of RNA and protein</h3>
<ul>
<li>~150 proteins</li>
<li>5 small nuclear RNAs(snRNAs) U1, U2, U4, U5, U6</li>
<li>RNA complexed with proteins. Small nuclear ribonuclease proteins(snRNPs)</li>
<li>**RNAs are thought to do the catalysis</li>
</ul>
<p><img src="media/16677507862746/16677798969995.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /><br />
In splicesome, RNA only do the catalysis, so there is only a little white in splicesome.</p>
<h2><a id="overview-of-spliceosome-mediated-splicing-reaction" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Overview of Spliceosome-mediated Splicing Reaction</h2>
<ul>
<li>U1 binds the 5' splice site</li>
<li>U2 binds the branch site</li>
<li>Together with other proteins</li>
</ul>
<p><img src="media/16677507862746/16677800288074.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<ul>
<li>
<p>Then snRNPs: U4, U5, U6 will come to recruit</p>
</li>
<li>
<p>U1 leaves</p>
</li>
<li>
<p>Now there is a <strong>loop</strong> in the intron.<br />
<img src="media/16677507862746/16677800810743.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
</li>
<li>
<p>U2 and U6 snRNAs help catalyze the first transesterification reaction</p>
</li>
<li>
<p>U4 leave</p>
</li>
</ul>
<p><img src="media/16677507862746/16677802869865.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<ul>
<li>
<p>U5 helps bring the exons together<br />
<img src="media/16677507862746/16677803169112.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
</li>
<li>
<p>Second transesterification reaction produces joined exons.</p>
</li>
</ul>
<h2><a id="spliceosome-mediated-splicing-reaction" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Spliceosome-mediated splicing reaction</h2>
<h3><a id="1-u1-binds-to-the-5-splice-site-u2-binds-to-the-branch-point" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>1. U1 binds to the 5' splice site; U2 binds to the branch point</h3>
<p><img src="media/16677507862746/16677804347449.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<p><em>RNA-RNA hybrids mediate splicing specificity.</em></p>
<h3><a id="2-the-u4-u5-u6-trimeric-snrnp-displaces-u1-at-the-5-splice-site-then-u4-dissociates" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>2. The U4-U5-U6 trimeric snRNP displaces U1 at the 5' splice site, then U4 dissociates</h3>
<p><img src="media/16677507862746/16677819010073.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<p><em>Extensive base pairing ensures specificity during splicing.</em></p>
<h3><a id="3-u6-and-u2-cataluze-the-attack-of-the-branchpoint-on-the-5-splice-site" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>3. U6 and U2 cataluze the attack of the branchpoint on the 5' splice site</h3>
<p><img src="media/16677507862746/16677820299751.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<h3><a id="4-the-5-splice-site-attacks-the-3-splice-site-completing-the-reaction" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>4. The 5' splice site attacks the 3' splice site, completing the reaction</h3>
<p><img src="media/16677507862746/16677821154979.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<h2><a id="other-types-of-intron-splicing-mechanisms" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Other Types of Intron Splicing Mechanisms</h2>
<h3><a id="group-ii-self-splicing" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Group II Self-splicing</h3>
<blockquote>
<p>Found in bacteria, fungi, organelles<br />
Catalyze their own splicing<br />
Use a mechanism similar to the spliceosome, but only use RNA(no proteins)</p>
</blockquote>
<p><img src="media/16677507862746/16677824144824.jpg" alt="" class="mw_img_center" style="width:300px;display: block; clear:both; margin: 0 auto;" /></p>
<h3><a id="group-i-self-splicing" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Group I Self-splicing</h3>
<p><img src="media/16677507862746/16677824400870.jpg" alt="" class="mw_img_center" style="width:300px;display: block; clear:both; margin: 0 auto;" /></p>
<p>Instead of Ademine, here is a guanine.<br />
Also, the intron becomes linear.</p>
<h3><a id="self-splicing-introns-have-complex-secondary-structures" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Self-splicing introns have complex secondary structures</h3>
<p><img src="media/16677507862746/16677825115278.jpg" alt="-c500 " /></p>
<h1><a id="mrna-splicing" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>mRNA Splicing</h1>
<h2><a id="ways-to-excise-mrna" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Ways to Excise mRNA</h2>
<ol>
<li>Proteins</li>
<li>Ribonucleoproteins</li>
<li>Self-excising</li>
</ol>
<h2><a id="transesterificatioin-reactions" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Transesterificatioin Reactions</h2>
<p><img src="media/16677507862746/16678381519553.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<p>In which two phosphodiester bond is broken. And replaced with an energetically equivalent phosphodiester bond without net input of energy.</p>
<h2><a id="mechanism-of-spliceosome-mediated-splicing" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Mechanism of Spliceosome-mediated Splicing</h2>
<blockquote>
<p>Splicing in Eukaryotes is catalyzed by the splicesome. <strong>A large ribonucleoprotein machine.</strong></p>
</blockquote>
<p>The spliceosome is composed of five individual spall ribonucleoproteins. U1, U2, U4, U5 and U6.</p>
<h3><a id="consensus-sequence" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Consensus Sequence</h3>
<ol>
<li>GU of the 5'</li>
<li>AG at the 3'</li>
<li>A branch point nucleotide A 10-20 upstream of 3'  site.</li>
</ol>

                  </article>
                  <div class="comments-wrap">
                    <div class="share-comments">
                      

                      
                        <div id="disqus_thread"></div>
                      

                      
                    </div>
                  </div><!-- end comments wrap -->
              </div>
            </div><!-- end columns -->
      </div><!-- end container -->
    </section>



    <footer class="footer">
        <div class="content has-text-centered">
          <p>
              Copyright &copy; 2019
              Powered by <a target="_blank" href="http://www.mweb.im">MWeb</a>,&nbsp; 
              Theme used <a target="_blank" href="https://bulma.io/">Bulma CSS</a>.
          </p>
        </div>
      </footer>



  
















<script type="text/javascript">
    var disqus_shortname = 'chiyuanye'; 

    (function() {
        var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
        dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
        (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
    })();
</script>

<script type="text/javascript">
var disqus_shortname = 'chiyuanye'; 

(function () {
var s = document.createElement('script'); s.async = true;
s.type = 'text/javascript';
s.src = '//' + disqus_shortname + '.disqus.com/count.js';
(document.getElementsByTagName('HEAD')[0] || document.getElementsByTagName('BODY')[0]).appendChild(s);
}());
</script>
  
    




  </body>
</html>
