<!DOCTYPE html>
<html lang="zh">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>
      
    Assessment2 Review - Yuanye Chi's Blog
    
    </title>
    <link rel="shortcut icon" href="https://i.loli.net/2019/02/20/5c6cf0555ffe6.jpeg" type="image/png" />

    
    
    <link href="atom.xml" rel="alternate" title="Yuanye Chi's Blog" type="application/atom+xml">
    <link rel="stylesheet" href="asset/css/style.min.css">
    <link rel="stylesheet" href="asset/css/doc.css">
    <script src="asset/app.js"></script>
</head>
  <body>
    <section class="hero">
      <div class="hero-head">
          <nav class="navbar" role="navigation" aria-label="main navigation">
              <div class="container">
              <div class="navbar-brand">
                
                <a target="_self" class="navbar-item " href="index.html">Home</a>
                
                <a target="_self" class="navbar-item " href="archives.html">Archives</a>
                
                <a target="_self" class="navbar-item " href="about.html">About</a>
                
                <a target="_self" class="navbar-item " href="https://chiyuanye-com-old-site.onrender.com/">OldSite</a>
                

                <a role="button" id="navbarSNSRssSwitchBtn" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navbarSNSRssButtons">
                  <span aria-hidden="true"></span>
                  <span aria-hidden="true"></span>
                  <span aria-hidden="true"></span>
                </a>
              </div>
            
              <div id="navbarSNSRssButtons" class="navbar-menu">
                <div class="navbar-start">
                  
                </div>
            
                <div class="navbar-end">
                  <div class="navbar-item">
                    <!--buttons start-->
                    <div class="buttons">
                      
                        
                        
                        
                        <a href="https://github.com/tsiye" target="_blank" title="github">
                            <span class="icon is-large has-text-grey-darker">
                               <svg class="svg-inline--fa fa-github fa-w-16 fa-lg" aria-hidden="true" focusable="false" data-prefix="fab" data-icon="github" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512" data-fa-i2svg=""><path fill="currentColor" d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"></path></svg><!-- <i class="fab fa-github fa-lg"></i> -->
                            </span>
                          </a>
                        
                        
                      
                      <a href="atom.xml" target="_blank" title="RSS">
                          <span class="icon is-large has-text-black-bis">
                              <svg class="svg-inline--fa fa-rss fa-w-14 fa-lg" aria-hidden="true" focusable="false" data-prefix="fas" data-icon="rss" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" data-fa-i2svg=""><path fill="currentColor" d="M128.081 415.959c0 35.369-28.672 64.041-64.041 64.041S0 451.328 0 415.959s28.672-64.041 64.041-64.041 64.04 28.673 64.04 64.041zm175.66 47.25c-8.354-154.6-132.185-278.587-286.95-286.95C7.656 175.765 0 183.105 0 192.253v48.069c0 8.415 6.49 15.472 14.887 16.018 111.832 7.284 201.473 96.702 208.772 208.772.547 8.397 7.604 14.887 16.018 14.887h48.069c9.149.001 16.489-7.655 15.995-16.79zm144.249.288C439.596 229.677 251.465 40.445 16.503 32.01 7.473 31.686 0 38.981 0 48.016v48.068c0 8.625 6.835 15.645 15.453 15.999 191.179 7.839 344.627 161.316 352.465 352.465.353 8.618 7.373 15.453 15.999 15.453h48.068c9.034-.001 16.329-7.474 16.005-16.504z"></path></svg><!-- <i class="fas fa-rss fa-lg"></i> -->
                          </span>
                      </a>
                    </div>
                    <!--buttons end-->

                  </div>
                </div>
                </div>
              </div>
            </nav>
      </div>

 <div class="hero-body ct-body"></div>
      
    </section>
    <section class="ct-body">
      <div class="container">
          <div class="columns is-variable bd-klmn-columns is-4 is-centered">
              <div class="column is-four-fifths">
                  <div class="post-body single-content">
                    
                    <h1 class="title">
                            Assessment2 Review   
                      </h1>
                     
                    
                      <div class="media">
                            
                            <figure class="media-left">
                              <p class="image is-48x48">
                                
                                  <img class="is-rounded" src="https://i.loli.net/2019/02/20/5c6cf0e7a42f9.jpg">
                                
                              </p>
                            </figure>
                            
                            <div class="media-content">
                              <div class="content">
                                <p>
                                 <span class="date">2022/10/29 20:13 PM</span>
                                  <span class="tran-posted-in">posted in</span>&nbsp; 
                                  
                                      <span class="posted-in"><a href='BIO105.html'>BIO105</a></span>
                                         
                                  
                                    &nbsp;&nbsp;<a href="16670888138748.html#disqus_thread"><span class="tran-disqus-comments">comments</span></a>
                                  

                                   
                                      
                                  <br />
                                  <span class="tran-tags">Tags:</span>&nbsp;
                                  
                                    <a class="tag is-link is-light" href='tag_BIO-105.html'>#BIO-105</a>
                                     

                                </p>
                              </div>
                            </div>
                         
                    </div>
                </div>
                  <article class="markdown-body single-content">
                    <h2><a id="overview" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Overview</h2>
<ul>
<li class="task-list-item"><input type="checkbox" checked="" disabled="" /> 1. Evaluate the experimental evidence for semiconservative, semidiscontinuous, and bi-</li>
<li class="task-list-item"><input type="checkbox" checked="" disabled="" /> directional DNA replication.</li>
<li class="task-list-item"><input type="checkbox" checked="" disabled="" /> 2. Draw a bi-directional replication fork, showing 5’ and 3’ ends, and identify the location of bacterial replication proteins relative to the fork.</li>
<li class="task-list-item"><input type="checkbox" checked="" disabled="" /> 3. Describe how the structures of the proteins involved in DNA replication help to support their roles.</li>
<li class="task-list-item"><input type="checkbox" checked="" disabled="" /> 4. Explain how the trombone model of DNA replication allows for coordination of leading and lagging strand synthesis in bacteria.</li>
<li class="task-list-item"><input type="checkbox" checked="" disabled="" /> 5. Compare and contrast mechanisms of replication initiation and termination in bacteria and eukaryotes.
<ul>
<li>The main difference between prokaryotic and eukaryotic DNA replication is that prokaryotic DNA replication occurs through a single origin of replication whereas eukaryotic DNA replication occurs through multiple replication origins.</li>
</ul>
</li>
<li class="task-list-item"><input type="checkbox" checked="" disabled="" /> 6. Identify characteristics of replication origins and explain how they are identified using both genetic and genomic methods.</li>
<li class="task-list-item"><input type="checkbox" checked="" disabled="" /> 7. Explain how replication is temporally regulated and coordinated with the cell cycle in eukaryotes.</li>
<li class="task-list-item"><input type="checkbox" checked="" disabled="" /> 8. Give examples of how water, oxygen, and chemicals can damage DNA.</li>
<li class="task-list-item"><input type="checkbox" checked="" disabled="" /> 9. Match different types of DNA damage with the corresponding repair system.</li>
<li class="task-list-item"><input type="checkbox" checked="" disabled="" /> 10. Describe the molecular mechanisms and identify key proteins in the mismatch repair, base excision repair, and nucleotide excision repair pathways.</li>
<li class="task-list-item"><input type="checkbox" checked="" disabled="" /> 11. Give two examples of mechanisms by which cells tolerate DNA damage during replication.</li>
<li class="task-list-item"><input type="checkbox" checked="" disabled="" /> 12. Diagram the steps by which non-homologous end joining, homologous recombination with crossovers, and synthesis-dependent strand annealing can be used to repair a DNA double-strand break.</li>
<li class="task-list-item"><input type="checkbox" checked="" disabled="" /> 13. Name the proteins involved in homologous recombination in bacteria and describe their roles.</li>
<li class="task-list-item"><input type="checkbox" checked="" disabled="" /> 14. Give examples of how defects in DNA repair systems can cause human diseases, including cancer.</li>
<li class="task-list-item"><input type="checkbox" checked="" disabled="" /> 15. Describe the origins of zinc finger nucleases, TALENS, and the CRISPR-Cas9 system and explain how these editing techniques have been adopted to promote double-strand break formation at specific sites in the genome.</li>
<li class="task-list-item"><input type="checkbox" checked="" disabled="" /> 16. Predict how double-strand break repair pathway choice will influence the outcomes of genome editing experiments.</li>
<li class="task-list-item"><input type="checkbox" checked="" disabled="" /> 17. Compare and contrast the mechanisms of homologous recombination, site-specific recombination, and transposition.</li>
<li class="task-list-item"><input type="checkbox" checked="" disabled="" /> 18. Given various starting DNA substrates, predict whether site-specific recombination will result in insertion, deletion, or inversion of DNA segments.</li>
<li class="task-list-item"><input type="checkbox" checked="" disabled="" /> 19. Describe the mechanisms of Cre-lox and FLP-FRT mediated site-specific recombination. Give examples of how these are used to edit genomes.</li>
<li class="task-list-item"><input type="checkbox" checked="" disabled="" /> 20. List the 3 main classes of transposons, explain how they transpose, and give examples of each.</li>
<li class="task-list-item"><input type="checkbox" checked="" disabled="" /> 21. Explain how transposons can be used to create transgenic organisms and to mutate genomes.</li>
</ul>
<h2><a id="enzymes" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Enzymes</h2>
<h3><a id="dna-replication-synthesis" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>DNA Replication Synthesis</h3>
<ol>
<li>DnaB helicase</li>
<li>RNA primase(create primer)</li>
<li>Polymerase III (do synthesis based on primer)</li>
<li>Polymerase I (Remove primer, exonuclease)</li>
<li>Ligase</li>
<li>Beta sliding clamp.</li>
</ol>
<h3><a id="dna-replication-initiation" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>DNA Replication Initiation</h3>
<h4><a id="bacteria" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Bacteria</h4>
<ol>
<li>Initiator: bind at the replicator(near A-T rich and easily melted area)</li>
<li>DnaA: It goes to wrap the DNA around it.</li>
<li>HU: is going to cause the formation of an open-complex.</li>
<li>DnaC: helicase loader complex</li>
<li>DnaB helicase</li>
<li>SeqA: Bind to hemi-methylated DNA to prevent combination of dam methylase</li>
<li>Dam methylase: doing methylation.</li>
</ol>
<h3><a id="dna-replication-termination" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>DNA Replication Termination</h3>
<h4><a id="bacteria" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Bacteria</h4>
<ol>
<li>Tus: bind to ter site to prevent replication in one direction.</li>
</ol>
<h3><a id="mismatch-repair" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Mismatch Repair</h3>
<h4><a id="bacteria" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Bacteria</h4>
<ol>
<li>MutS: detect hemi-methylated strand</li>
<li>MutS-MutL complex: scan DNA bidirectionally, forming a loop. It is looking for the GATC hemi-methylated sides.</li>
<li>MutH: recruited by MutS-MutL to cleave the newly synthesized unmethylated GATC sequence.</li>
<li>Helicase(UvrD)</li>
<li>Endonuclease/polymerase I</li>
<li>Polymerase III</li>
<li>Ligase</li>
</ol>
<h4><a id="eukaryote" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Eukaryote</h4>
<ol>
<li>ORC: Origin recognition complex</li>
<li>Mcm2-7: DNA helicase</li>
<li>Cdt1, Cdc6: Helicase Loader.</li>
<li>CDK, DDK: protein kinase like DDK and CDK will give helicase the signal to do unwinding.</li>
</ol>
<h3><a id="direct-reversal" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Direct Reversal</h3>
<p>Methyltransferase: reverse Methyl Group</p>
<h3><a id="base-excision-repair" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Base Excision Repair</h3>
<h4><a id="bacteria" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Bacteria</h4>
<ol>
<li>Glycoslyase: produce a abasic site</li>
<li>AP endonuclease: remove the abasic site</li>
<li>Polymerase I: synthesize the removed sites</li>
<li>Ligase</li>
</ol>
<h4><a id="eukarytotes" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Eukarytotes</h4>
<ol>
<li>Glycoslyase: produce a abasic site</li>
<li>AP endonuclease: remove the abasic site</li>
<li>Polymerase I: synthesize the removed sites(long patch)</li>
<li>Polymerase beta: short patch</li>
<li>Ligase</li>
</ol>
<h3><a id="nucleotide-excision-repair" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Nucleotide Excision Repair</h3>
<h4><a id="bacteria" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Bacteria</h4>
<ol>
<li>UvrA/B: recognizes damage</li>
<li>UrvC: exinuclease</li>
<li>UvrD: helicase</li>
<li>Polymerase I</li>
<li>Ligase</li>
</ol>
<h4><a id="eukarytotes" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Eukarytotes</h4>
<ol>
<li>XPC/XPB/XPD:  recognize</li>
<li>RPA/XPF/XPG: exinuclease</li>
<li>Helicase</li>
<li>Polymerase</li>
<li>Ligase</li>
</ol>
<h3><a id="translesion-synthesis" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Translesion Synthesis</h3>
<ol>
<li>Polymerase III</li>
<li>beta clamp</li>
<li>Pol IV or Pol V (TLS polymerase)</li>
</ol>
<h3><a id="fork-regression-fork-stalls" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Fork Regression(fork stalls)</h3>
<ol>
<li>Some proteins use ATP hydrolysis to reverse the fork</li>
<li>Nuclease</li>
</ol>
<h3><a id="double-strand-break-repair" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Double-Strand Break Repair</h3>
<ol>
<li>Ku70-Ku80: proteins bind ends and protect</li>
<li>DNA-PKcs and Artemis: Artemis is a nuclease</li>
<li>XLF, XRCC4, Ligase IV</li>
</ol>
<h3><a id="homologous-recombinatino" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Homologous Recombinatino</h3>
<h4><a id="bacteria" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Bacteria</h4>
<p>RecBCD, RecA, RuvAB complex, RuvC</p>
<h4><a id="eukaryote" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Eukaryote</h4>
<p>helicases, many exo and endonucleases, Rad51, Rad52, Brca2, RecQ helicases, many different resolvases<br />
<img src="media/16670888138748/16673385399887.jpg" alt="" class="mw_img_center" style="width:600px;display: block; clear:both; margin: 0 auto;" /></p>
<h3><a id="crispr" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>CRISPR</h3>
<p>Cas9 nuclease</p>
<h3><a id="cssr" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>CSSR</h3>
<p>Recombinase</p>
<h3><a id="transposition" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Transposition</h3>
<ol>
<li>Transposase</li>
<li>DNA polymerase</li>
<li>DNA ligase</li>
</ol>
<h2><a id="crisprcas9" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>CRISPR/Cas9</h2>
<p>The mechanism of CRISPR/Cas-9 genome editing can be generally divided into three steps: recognition, cleavage, and repair.</p>
<h3><a id="1-recognition" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>1. Recognition</h3>
<p>The designed sgRNA directs Cas-9 and recognizes the target sequence in the gene of interest through its 5ʹcrRNA complementary base pair component. The Cas-9 protein remains inactive in the absence of sgRNA. The Cas-9 nuclease makes double-stranded breaks (DSBs) at a site 3 base pair upstream to PAM.14 PAM sequence is a short (2–5 base-pair length) conserved DNA sequence downstream to the cut site and its size varies depending on the bacterial species. The most commonly used nuclease in the genome-editing tool, Cas-9 protein recognizes the PAM sequence at 5ʹ-NGG-3ʹ (N can be any nucleotide base).</p>
<h3><a id="2-cleavage" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>2. Cleavage</h3>
<p>Once Cas-9 has found a target site with the appropriate PAM, it triggers local DNA melting followed by the formation of RNA-DNA hybrid, but the mechanism of how Cas-9 enzyme melts target DNA sequence was not clearly understood yet. Then, the Cas-9 protein is activated for DNA cleavage. HNH domain cleaves the complementary strand, while the RuvC domain cleaves the non-complementary strand of target DNA to produce predominantly blunt-ended DSBs.</p>
<h3><a id="3-repair" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>3. Repair</h3>
<p>Finally, the DSB is repaired by the host cellular machinery.</p>
<h2><a id="assays" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Assays</h2>
<h3><a id="incorporation-assays" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Incorporation Assays</h3>
<p><img src="media/16670888138748/16673202935806.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<h3><a id="primer-extension-assays" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Primer Extension Assays</h3>
<p><img src="media/16670888138748/16673203238573.jpg" alt="" class="mw_img_center" style="width:500px;display: block; clear:both; margin: 0 auto;" /></p>
<h2><a id="summary-of-mechanisms-for-dealing-with-dna-damage" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Summary of mechanisms for dealing with DNA damage</h2>
<h3><a id="1-base-mismatches-base-damage" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>1. Base mismatches/ base damage</h3>
<ul>
<li>Base excision repair removes  base damage</li>
<li>Nucleotide excision repair removes damage that disttorts the helix</li>
<li>Mismatch repair corrects mispaired bases due to erroneous insertion during DNA replication</li>
</ul>
<h3><a id="2-bypass-damage" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>2. Bypass Damage</h3>
<p>Y-family DNA polymerase replicate past DNA damage</p>
<h3><a id="3-strand-breaks" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>3. Strand breaks</h3>
<ul>
<li>Non-homologous end joining repairs</li>
<li>Homologous recombination repairs</li>
</ul>
<h2><a id="review-different-genome-editing-systems" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>Review: Different Genome Editing Systems</h2>
<ol>
<li>Site-specific recombination (Cre-lox, FLP-FRT)</li>
<li>Transposons (Tn10, Sleeping Beauty)</li>
<li>Genome editing nucleases (ZFNs, TALENs, CRISPR)</li>
</ol>

                  </article>
                  <div class="comments-wrap">
                    <div class="share-comments">
                      

                      
                        <div id="disqus_thread"></div>
                      

                      
                    </div>
                  </div><!-- end comments wrap -->
              </div>
            </div><!-- end columns -->
      </div><!-- end container -->
    </section>



    <footer class="footer">
        <div class="content has-text-centered">
          <p>
              Copyright &copy; 2019
              Powered by <a target="_blank" href="http://www.mweb.im">MWeb</a>,&nbsp; 
              Theme used <a target="_blank" href="https://bulma.io/">Bulma CSS</a>.
          </p>
        </div>
      </footer>



  
















<script type="text/javascript">
    var disqus_shortname = 'chiyuanye'; 

    (function() {
        var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
        dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
        (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
    })();
</script>

<script type="text/javascript">
var disqus_shortname = 'chiyuanye'; 

(function () {
var s = document.createElement('script'); s.async = true;
s.type = 'text/javascript';
s.src = '//' + disqus_shortname + '.disqus.com/count.js';
(document.getElementsByTagName('HEAD')[0] || document.getElementsByTagName('BODY')[0]).appendChild(s);
}());
</script>
  
    




  </body>
</html>
